supplementary/alica_ros_turtlesim/ros1/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ catkin_package(
 ###########
 
 include_directories(
+    ../include
```

Comment:
seems wrong

Reply:
This adds the include folder of the parent directory alica_ros_turtlesim, which containts the turtle_interfaces.hpp file. Both ros1 and ros2 have derived classes from it so they need to include it